tensorflow/core/kernels/string_split_op.cc

```diff
@@ -118,10 +117,50 @@ class StringSplitOp : public OpKernel {
     }
   }
 
- private:
+ protected:
```

Comment:
Let's make `skip_empty_` a private member of the subclasses. Doesn't seem like it us used in the base class and it is only set by the subclass constructors, so it will be cleaner to move it down there?

Reply:
Done.